extensions/amp-animation/0.1/css-expr-impl.jison

```diff
@@ -0,0 +1,319 @@
+/**
+ * https://www.w3.org/TR/CSS22/grammar.html
+ */
+
+%lex
+%option case-insensitive
+
+A         [Aa]
+B         [Bb]
+C         [Cc]
+D         [Dd]
+E         [Ee]
+F         [Ff]
+G         [Gg]
+H         [Hh]
+I         [Ii]
+J         [Jj]
+K         [Kk]
+L         [Ll]
+M         [Mm]
+N         [Nn]
+O         [Oo]
+P         [Pp]
+Q         [Qq]
+R         [Rr]
+S         [Ss]
+T         [Tt]
+U         [Uu]
+V         [Vv]
+W         [Ww]
+X         [Xx]
+Y         [Yy]
+Z         [Zz]
+
+num       [+-]?[0-9]+(\.[0-9]+)?([eE][+\-]?[0-9]+)?|[+-]?"."[0-9]+([eE][+\-]?[0-9]+)?
```

Comment:
Fixed. These and all others actually come directly from CSS spec (https://www.w3.org/TR/CSS22/grammar.html) and such.